indeex.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheet.css">
+    <header>
+        <!-- section which is not needed -->
+    <div class="logo"><!-- logo will be put here --></div>
```

Comment:
I cannot find a logo class in your css file?

Reply:
No need for that class for now, DOM is empty for now.

Reply:
resolved!